index.html

```diff
@@ -0,0 +1,163 @@
+				</li>
+			</ul>
+		</div>
+		<div class="product-details-container">
+			<p class="product-text">Back to <a href="#" class="for-text-links">Woman<a/></p>
+			<h2 class="name-product">The Atelier Tailored Coat</h2>
+			<div class="rating">
+				<span class="rating-stars">☆</span><span class="rating-stars">☆</span><span class="rating-stars">☆</span><span class="rating-stars">☆</span><span class="rating-stars">☆</span>
+				<p class="product-rating-revies">3 Review(s) <span class="add-class-review">add a review</span></p>
+			</div>
+			<p class="product-price">&#36;499.00</p>
+			<ul class="product-secondary-infostore">
+				<li class="product-secondary-infostore">Availability :<span class="product-secondary-infostore-status">In stock</li>
+				<li class="product-secondary-infostore">Product code :<span class="product-secondary-infostore-status"> &#35;4657</li>
+				<li class="product-secondary-infostore">Tags:
+					<a href="#" class="for-text-links">Fashion</a>
+					<a href="#" class="for-text-links">Hood</a>
+					<a href="#" class="for-text-links">Classic</a>
+				</li>
+			</ul>
+			<p class="product-description">Sleek, polished, and boasting an impeccably modern fit, this blue, 2-but-
+			ton Lazio suit features a notch lapel, flap pockets, and accompanying flat
+			front trousers—all in pure wool by Vitale Barberis Canonico.
+			</p>
+			<ul class="product-description-info">
+				<li>Dark blue suit for a tone-on-tone look</li>
+				<li>Regular fit</li>
+				<li>100% Cotton</li>
+				<li>Free shipping with 4 days delivery</li>
+			</ul>
+			<form method="post" action="product_order.php" name="product-order" class="product-order">
+				<label for="color-selector" class="color-selector" >Color</label>
+				<select class="product-order-selector" name="product-order-color">
+					<option value="" disabled selected>Select Color</option>
+					<option value="Black">Black</option>
+					<option value="White">White</option>
+					<option value="Green">Green</option>
+				</select>
+				<label for="size-selector" class="size-selector" >Size</label>
+				<select class="product-order-selector" name="product-order-color">
+					<option value="" disabled selected>Select Size</option>
+					<option value="M">M</option>
+					<option value="L">L</option>
+					<option value="XL">XL</option>
+					<option value="XXL">XXL</option>
+				</select>
+				<label for="quantity-selector" class="quantity-selector" >QTY</label>
+				<input type="number" name="quantity" class="product-order-quantity" min="1" max="10" value="1">
+				<div class=button-reset-for-order>
+					<button type="reset">Clear Selection</button>
+				</div>
+			</form>
+			<div class="adding-list-bar">
+				<ul class="adding-list-bar-button">
+					<li><a href="#" id="add-to-cart">add to cart</a></li>
+					<li><a href="#" id="add-to-wishlist">&hearts;  add to wishlist</a></li>
+				</ul>
+			</div>
+			<div class="social-tray">
+				<div class="social-tray-text">
```

Comment:
1. no need for p
2. this is label for the whole block, use this word in classname